docs/developers/contributing.rst

```diff
@@ -335,11 +335,9 @@ Release procedure
 
 Pre-release
 """""""""""
-1. Make sure that all pull requests which will be
-   included in the release have been properly documented in
-   :file:`docs/release-notes.rst`.
-2. Rename the "Unreleased" section heading in :file:`docs/release-notes.rst`
-   to the version you are about to release.
+1. Make sure that all pull requests which will be included in the release
+   have been properly documented as changelog files in :file:`changes`.
```

Comment:
can you elaborate on / provide examples for what structure a changelog file should have?

Reply:
Does this line in `changes/README.md` answer your question?

> Inside the file, please write a short description of what you have changed, and how it impacts users of `zarr-python`.

Reply:
yep, that's nice and clear